osidb/api_views.py

```diff
@@ -345,6 +354,75 @@ def create(self, request, *args, **kwargs):
         return response
 
 
+@include_meta_attr_extend_schema_view
+@include_exclude_fields_extend_schema_view
+@extend_schema_view(
+    create=extend_schema(
+        request=FlawAcknowledgmentPostSerializer,
+    ),
+    update=extend_schema(
+        request=FlawAcknowledgmentPutSerializer,
+    ),
+)
+class FlawAcknowledgmentView(ModelViewSet):
+    serializer_class = FlawAcknowledgmentSerializer
+    # As long as SRTNotesBuilder.generate_acknowledgments() builds from FlawMeta,
+    # POST, DELETE and PUT don't work.
+    # TODO: Resolve in a follow-up to PR 265.
+    http_method_names = get_valid_http_methods(
+        ModelViewSet, excluded=["post", "delete", "put"]
+    )
+    permission_classes = [IsAuthenticatedOrReadOnly]
+    filterset_class = FlawAcknowledgmentFilter
+
+    def destroy(self, request, *args, **kwargs):
+        bz_api_key = request.META.get("HTTP_BUGZILLA_API_KEY")
+        if not bz_api_key:
+            raise ValidationError({"Bugzilla-Api-Key": "This HTTP header is required."})
+        instance = self.get_object()
+        self.perform_destroy(instance, bz_api_key=bz_api_key)
+        return Response(status=HTTP_200_OK)
+
+    def perform_destroy(self, instance, bz_api_key):
+        """
+        override the default behavior to proxy the delete to Bugzilla
+        """
+        flaw = instance.flaw
+        instance.delete()
+        flaw.save(bz_api_key=bz_api_key)
+
+    def get_flaw(self):
+        """
+        Get flaw object instance.
+        Similar to FlawView.get_object().
```

Comment:
Please be more descriptive in docstrings (but still brief). I mean - Which flaw should be gotten? The one associated with a given flaw acknowledgment. If a docstring is too general, it has no value for people who read it.

You can also omit `Similar to FlawView.get_object()` as it is useless.

Reply:
Fixed in commits 09866d24b5b72af9f19ea30c518961a04686bd1b and eaaa28d2c2b1ea423c782b4745faa28b97bf3e7d